### ISSUE TITLE
Accessibility information: visual adjustments

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/readium-desktop#3517]
visual adjustments: when `is_fixed_layout` is true, the technique says to display “Appearance cannot be modified”, but the component currently uses the same localized key as the modifiable case, so fixed-layout publications is labeled incorrectly.

https://github.com/edrlab/thorium-reader/blob/198b8fb33fd606e11d741dd50ec3a6a386517df1/src/renderer/common/components/dialog/publicationInfos/PublicationInfoA11y2.tsx#L94

https://www.w3.org/community/reports/publishingcg/CG-FINAL-epub-techniques-20251222/#visual-adjustments
